### PR TITLE
fix(framework): strip whitespaces around < and >

### PR DESCRIPTION
**Pull request by fifoosid**

### CODE REVIEW

File: packages/tools/lib/hbs2lit/src/compiler.js [hunk 1]
```diff
@@ -5,7 +5,12 @@ const includesReplacer = require("./includesReplacer");
 const svgProcessor = require("./svgProcessor");
 
 const removeWhiteSpaces = (source) => {
-	return source.replace(/\n+/g, "").replace(/\s+</g, "<").replace(/}}\s+{{/g, "}}{{").replace(/\t+/g, " ");
+	return source
+		.replace(/\n+/g, "")
+		.replace(/\s*<\s*/g, "<")
+		.replace(/(!?[a-z\/\"])>\s+/g, "$1>") // skips striping of =>(arrow functions) and -->(comments). Strips only when needed.
```
Comment (vladitasev) [inline packages/tools/lib/hbs2lit/src/compiler.js:11]: Do we have arrow functions in any of the hbs files?
Also, why do we need to do extra checks for comments: for example ` <!-- asdf --> ` can safely be stripped of all whitespace around `<` and `>` so I don't see a reason why we don't use: `.replace(/\s*<\s*/g, "<")` and `.replace(/\s*>\s*/g, ">")` only

File: packages/main/src/TextArea.hbs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 				{{this.text}}
 
 				{{#unless this.last}}
-					<br/>
+					\n
```
Comment (vladitasev) [inline packages/main/src/TextArea.hbs:15]: I think `<br>` is the right one here, when you write HTML you never type `\n` for a new line.

File: packages/tools/lib/hbs2lit/src/compiler.js (hunk 1 above)
Comment (fifoosid) [reply packages/tools/lib/hbs2lit/src/compiler.js:11]: Done

File: packages/main/src/TextArea.hbs (hunk 2 above)
Comment (fifoosid) [reply packages/main/src/TextArea.hbs:15]: Done

**Pull request merged**